### PR TITLE
Media Library: small UI enhancements

### LINKED ISSUE #2732: Small UI improvements for media  library
## Feature Description

As per @samitron7's feedback:

* Change the color of the circle in the more menu to `#2F3449`. Black is too striking
* Change the color of the circle in the media edit button to `#2F3449`.
* Remove the play button. This isn't needed since we have the time duration.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Implementation Brief

* Remove icon
* Change "more" icon so color can be set via CSS and is not hardcoded in the SVG

### PR DESCRIPTION
**Pull request by swissspidy**
## Summary

Addresses some small UX feedback in the media library.

## User-facing changes

* Change the color of the circle in the more menu to #2F3449.
* Removed the play button.

Screenshot 2020-06-25 at 14 59 34

## Testing Instructions

* Verify that there's no play icon anymore
* "more" menu bg color changed

---

Fixes #2732

### CODE REVIEW

File: assets/src/edit-story/components/library/panes/media/dropDownMenu.js [hunk 1]
```diff
@@ -39,6 +39,9 @@ const MoreButton = styled(More)`
   position: absolute;
   top: 8px;
   right: 8px;
+  background: ${({ theme }) => theme.colors.bg.v4};
```
Comment (miina) [inline assets/src/edit-story/components/library/panes/media/dropDownMenu.js:42]: Just to confirm: should the Media edit icon in the Accessibility panel also get a new background color or is it intended that it's only for the Media menu?
Screenshot 2020-06-25 at 15 53 55

(This icon will change within #1218 anyway though so perhaps it's ok to keep as is.)

Comment (samitron7) [reply assets/src/edit-story/components/library/panes/media/dropDownMenu.js:42]: @miina Yes the Media edit icon in the Accessibility panel should also get a new background color

Comment (swissspidy) [reply assets/src/edit-story/components/library/panes/media/dropDownMenu.js:42]: Done 👍

**Review by miina — APPROVED** [review]
LGTM

**Pull request merged**